thrust.diff

```diff
@@ -0,0 +1,32 @@
+diff --git a/third_party/thrust/thrust/detail/type_traits.h b/third_party/thrust/thrust/detail/type_traits.h
+index d147f832..60b57654 100644
+--- a/third_party/thrust/thrust/detail/type_traits.h
++++ b/third_party/thrust/thrust/detail/type_traits.h
+@@ -144,11 +144,11 @@ template<typename T> struct has_trivial_constructor
+       is_pod<T>::value
+ #if THRUST_HOST_COMPILER == THRUST_HOST_COMPILER_MSVC || \
+     THRUST_HOST_COMPILER == THRUST_HOST_COMPILER_CLANG
+-      || __has_trivial_constructor(T)
++      || __is_trivially_constructible(T)
```

Comment:
Do we think thrust has a bug? Any idea what changed on the CI; can you repro locally?

Reply:
they use a deprecated API: https://github.com/NVIDIA/thrust/issues/1743